src/test/rustdoc-ui/reference-link-reports-error-once.rs

```diff
@@ -0,0 +1,20 @@
+#![deny(broken_intra_doc_links)]
+
+/// Links to [a] [link][a]
+/// And also a [third link][a]
+/// And also a [reference link][b]
+///
+/// Other links to the same target should still emit error: [ref] //~ERROR unresolved link to
```

Comment:
Can you include the text of the item that failed so this is easier to read?
```suggestion
/// Other links to the same target should still emit error: [ref] //~ERROR unresolved link to `ref`
```